tests/mypy_test.py

```diff
@@ -209,7 +210,8 @@ def add_configuration(configurations: list[MypyDistConf], distribution: str) ->
     with Path("stubs", distribution, "METADATA.toml").open("rb") as f:
         data = tomli.load(f)
 
-    mypy_tests_conf = data.get("mypy-tests")
+    # TODO: This could be added to parse_metadata.py, but is currently unused
+    mypy_tests_conf: dict[str, dict[str, dict[str, dict[str, Any]]]] = data.get("mypy-tests", {})
```

Comment:
I'd just go with a simpler, less verbose type hint for now, tbh, since this is currently unused

```suggestion
    mypy_tests_conf: dict[str, Any] = data.get("mypy-tests", {})
```